diff --git a/node_modules/@firebase/firestore-compat/dist/src/index.d.ts b/node_modules/@firebase/firestore-compat/dist/src/index.d.ts
index 8b74715..d23f09f 100644
--- a/node_modules/@firebase/firestore-compat/dist/src/index.d.ts
+++ b/node_modules/@firebase/firestore-compat/dist/src/index.d.ts
@@ -49,7 +49,7 @@ declare module '@firebase/app-compat' {
 }
 
 import { FirebaseApp as FirebaseAppCompat } from "@firebase/app-compat";
-import { type WithFieldValue, type DocumentReference, type CollectionReference, type DocumentData, type Query, type EmulatorMockTokenOptions, type PersistenceSettings, type AggregateQuerySnapshot, type AggregateField, type DocumentSnapshot, type QuerySnapshot, type Firestore, type FirestoreSettings, type LoadBundleTask, type FirestoreError, type Unsubscribe, type SnapshotListenOptions, type QueryCompositeFilterConstraint, type QueryNonFilterConstraint, type QueryConstraint, type Transaction, type TransactionOptions, type PartialWithFieldValue, type SetOptions, type IndexConfiguration, type UpdateData, type FieldPath, type WriteBatch } from "@firebase/firestore";
+import { WithFieldValue, DocumentReference, CollectionReference, DocumentData, Query, EmulatorMockTokenOptions, PersistenceSettings, AggregateQuerySnapshot, AggregateField, DocumentSnapshot, QuerySnapshot, Firestore, FirestoreSettings, LoadBundleTask, FirestoreError, Unsubscribe, SnapshotListenOptions, QueryCompositeFilterConstraint, QueryNonFilterConstraint, QueryConstraint, Transaction, TransactionOptions, PartialWithFieldValue, SetOptions, IndexConfiguration, UpdateData, FieldPath, WriteBatch } from "@firebase/firestore";
 declare module "@firebase/firestore" {
     function addDoc<T>(reference: types.CollectionReference<T>, data: WithFieldValue<T>): Promise<DocumentReference<T>>;
     function clearIndexedDbPersistence(firestore: types.FirebaseFirestore): Promise<void>;
