diff --git a/node_modules/@firebase/auth-compat/dist/auth-compat/index.d.ts b/node_modules/@firebase/auth-compat/dist/auth-compat/index.d.ts
index dad4acb..2b71e07 100644
--- a/node_modules/@firebase/auth-compat/dist/auth-compat/index.d.ts
+++ b/node_modules/@firebase/auth-compat/dist/auth-compat/index.d.ts
@@ -50,7 +50,7 @@ declare module '@firebase/app-compat' {
 }
 
 import { FirebaseApp as FirebaseAppCompat } from "@firebase/app-compat";
-import { type User, type Unsubscribe, type ActionCodeInfo, type UserCredential, type Auth, type IdTokenResult, type MultiFactorError, type MultiFactorResolver, type PopupRedirectResolver, type Dependencies, type AuthCredential, type ApplicationVerifier, type ConfirmationResult, type AuthProvider, type MultiFactorUser, type NextOrObserver, type ErrorFn, type CompleteFn, type ActionCodeSettings, type Persistence, type PhoneAuthCredential } from "@firebase/auth";
+import { User, Unsubscribe, ActionCodeInfo, UserCredential, Auth, IdTokenResult, MultiFactorError, MultiFactorResolver, PopupRedirectResolver, Dependencies, AuthCredential, ApplicationVerifier, ConfirmationResult, AuthProvider, MultiFactorUser, NextOrObserver, ErrorFn, CompleteFn, ActionCodeSettings, Persistence, PhoneAuthCredential } from "@firebase/auth";
 declare module "@firebase/auth" {
     function applyActionCode(auth: types.FirebaseAuth, oobCode: string): Promise<void>;
     function beforeAuthStateChanged(auth: types.FirebaseAuth, callback: (user: User | null) => void | Promise<void>, onAbort?: () => void): Unsubscribe;
